lib/pi_piper/driver.rb

```diff
@@ -0,0 +1,114 @@
+module PiPiper
+  class Driver
+    
+    VERSION = '0.1.0'
+    ALLOW = []
+    API_VERSION = '0.1.0'
```

Comment:
What is this used for? The `PiPiper` version should be sufficient since this driver will be in lockstep with those releases. If someone needs a specific version, they can manage which version of `PiPiper` they want.